src/rune-bridge/src/ops/tests/await_confirmations.rs

```diff
@@ -0,0 +1,180 @@
+use std::str::FromStr;
+
+use bridge_did::error::Error;
+use did::H160;
+use ic_exports::ic_cdk::api::management_canister::bitcoin::{Outpoint, Utxo};
+use ic_exports::ic_kit::MockContext;
+use snapbox::{assert_data_eq, str};
+
+use crate::core::utxo_handler::test::TestUtxoHandler;
+use crate::core::utxo_handler::{RuneToWrap, UtxoHandlerError};
+use crate::ops::{tests, RuneBridgeOp};
+use crate::rune_info::{RuneInfo, RuneName};
+
+fn get_utxo() -> Utxo {
+    Utxo {
+        outpoint: Outpoint {
+            txid: vec![],
+            vout: 0,
+        },
+        value: 0,
+        height: 0,
+    }
+}
+
+fn get_to_wrap(count: usize) -> Vec<RuneToWrap> {
+    let mut result = vec![];
+    for _ in 0..count {
+        result.push(RuneToWrap {
+            rune_info: RuneInfo {
+                name: RuneName::from_str("A").unwrap(),
```

Comment:
I'm not sure if it's the same, but for coherence with real-case scenario, the rune name here should be randomized in order to have each runetoWrap different